docs/13-walkthroughs/3-add-logging.md

```diff
@@ -1,39 +1,59 @@
-# Add Logging
+# Add Logging to an Existing Zarf Cluster with Zarf Init
```

Comment:
Not a big thing, but this title being longer causes it to wrap @Jessy-Morris not sure if this is a style thing we want to document but this was something beaten into me in the Air Force to avoid (not necessarily saying that is right though)

![image](https://user-images.githubusercontent.com/3977569/227033178-c491e5b1-7644-4e19-b316-08f2069926d8.png)

Reply:
Made this shorter